06-raster-vector.qmd

```diff
@@ -472,9 +473,87 @@ axes[1][0].set_title('Count')
 axes[1][1].set_title('Aggregated capacity');
 ```
 
-...
+Another dataset based on California's polygons and borders (created below) illustrates rasterization of lines. There are three preliminary steps. First, we subset the California polygon:
 
-## Spatial vectorization
+```{python}
+california = us_states[us_states['NAME'] == "California"]
+california
+```
+
+Second, we "cast" the polygon into a `"MultiLineString"` geometry, using the `.boundary` property that `shapely` geometries have: 
+
+```{python}
+california_borders = california['geometry'].iloc[0].boundary
```

Comment:
Interesting syntax. Why `.iloc[0]` is the question that comes to mind, would have assumed `california['geometry'].boundary` would work.

Reply:
> Interesting syntax. Why `.iloc[0]` is the question that comes to mind, would have assumed `california['geometry'].boundary` would work.

These are two methods to do the same thing, one using `shapely` and the other using `geopandas`. I agree it's better to be consistent, now changed to `geopandas` same as in the preceding example
https://github.com/geocompx/geocompy/commit/d81c253a341d73ee8dd83baaa400ac7578170074